source/cloud-security/azure/activity-services/services/storage.rst

```diff
@@ -23,7 +23,7 @@ This section explains how to use the Azure portal to archive the Azure activity
 Configuring the Activity log export
 -----------------------------------
 
-1. To export the logs, search for the **Activity log** service. It can be found by typing "Activity" in the search engine. From there, access the **Audit Logs** section and click on **Export Data Settings**.
+1. To export the logs, search for the **Activity log** service. It can be found in the **Monitor** service menu or by typing "Activity" in the search engine. From there, click on **Export Data Settings**.
```

Comment:
I think this needs the following:

- Image addition/update to show how to find the logs. Do we mean **Monitor** or **Monitoring**. Maybe an image similar to this one
  ![imagen](https://github.com/wazuh/wazuh-documentation/assets/47069802/eea12fa9-c96c-414f-a573-ed409459fe36)

- The phrase "Search for the **Activty log** service" is too vague. We could rephrase the instruction into something more actionable, like
  - "Find the activity logs to export in one of these two ways:
    - Click...
    - Type..."

Reply:
Added in 566df6bf50b8a5efca2ad03047554e1ac4436d57.